docs/README.md

```diff
@@ -3,9 +3,23 @@
 
 ## Quick start 
 Binaries can be found [here](https://github.com/mintlayer/core/releases).
+
+Running a node also requires as input a chain specification.
+Currently a single chain specification for the testnet is provided, and can be downloaded using curl:
+
+```
+curl --proto '=https' -sSf 								\
+    https://raw.githubusercontent.com/mintlayer/core/master/assets/Testnet1Spec.json 	\
+    --output Testnet1Spec.json
+```
 Download and run:
 ```
-mintlayer-core --base-path data/my_first_ml_node --validator --rpc-external --rpc-methods Unsafe --chain=Testnet1Spec.json
+mintlayer-core 				\
+    --base-path data/my_first_ml_node 	\
+    --validator 			\
+    --rpc-external 			\
+    --rpc-methods Unsafe 		\
+    --chain=Testnet1Spec.json
```

Comment:
I never used the `--rpc-external` and `--rpc-methods Unsafe` options so at least for basic usage, these are not necessary. The question is whether users should pass them by default. Ideally, the command to run the node should be as simple as possible.

Reply:
If I understand correctly, the `rpc-external` flag binds to all network interfaces with IP 0.0.0.0, meaning that anyone can connect to the node from outside the network. And `rpc-methods Unsafe`, basically allows anyone connecting to the RPC to be an owner of the node, and hence, expose keys. Meaning: We're basically exposing everything to everyone and allowing anyone connecting to RPC to do whatever they want... what's the logic behind this?! Did we try not adding them and had problems?

Reply:
This is the command to run as I understood it from Amin.
@zorvan  What do you think of Lukas's and Sam's comments?

Reply:
They are right. Latest version of cli switches for user and production id reaped off from those.

Reply:
@zorvan  so what is the correct command? Should I simply remove --rpc-external and --rpc-methods Unsafe?

Reply:
As in every issue in programming, that depends on what you want to do. The right solution, in my opinion, is to remove these and tell the user that they will be binding to `localhost`, hence, no connections from the outside will be possible and explain that this is a safety procedure and using tunnels is usually better than opening to the outside, and then explain that unsafe methods will not work unless the user enables them.

Reply:
And let me explain that hacks with externally accessible RPC happened before: 

https://www.coindaily.co/2801-2/